GitHubCore.ps1

```diff
@@ -223,8 +223,15 @@ function Invoke-GHRestMethod
                 }
             }
 
+            # Temporarily Disable Progress Bar during Invoke-WebRequest
+            $tempProgressPreference = $ProgressPreference
```

Comment:
If `Invoke-WebRequest` fails/throws, we'll never restore this.  I think you should save the state before the `try` @ 199, and restore it in a newly added `finally` at the end of the function.

Reply:
As above.

Reply:
Done.